auth-web/src/services/org.services.ts

```diff
@@ -88,8 +97,19 @@ export default class OrgService {
     return axios.patch(`${ConfigHelper.getAuthAPIUrl()}/orgs/${orgIdentifier}/status`, { statusCode: 'REJECTED' })
   }
 
-  static async suspendOrg (orgIdentifier: number, statusCode: AccountStatus, suspensionReasonCode: string): Promise<AxiosResponse> {
-    return axios.patch(`${ConfigHelper.getAuthAPIUrl()}/orgs/${orgIdentifier}/status`, { statusCode: statusCode, suspensionReasonCode: suspensionReasonCode })
+  static async patchOrg (patchOrgPayload: PatchOrgPayload): Promise<AxiosResponse> {
+    switch (patchOrgPayload.action) {
+      case PatchActions.UPDATE_STATUS:
+        return axios.patch(`${ConfigHelper.getAuthAPIUrl()}/orgs/${patchOrgPayload.orgIdentifier}`, { statusCode: patchOrgPayload.statusCode,
+          suspensionReasonCode: patchOrgPayload.suspensionReasonCode,
+          action: patchOrgPayload.action })
+      case PatchActions.UPDATE_ACCESS_TYPE:
+        return axios.patch(`${ConfigHelper.getAuthAPIUrl()}/orgs/${patchOrgPayload.orgIdentifier}`, { accessType: patchOrgPayload.accessType,
+          action: patchOrgPayload.action })
+      default:
+        break
+    }
```

Comment:
since axios.patch is common ,move it outside of the method and just make the request body diffenrent?